### PR TITLE
feat: Create service account and roles

### PR DESCRIPTION
**Pull request by SamoKopecky**
References #23, #24 
- `ServiceAccount` and `serviceAccountName` is already created in #19 
- Create a `role` and a `role-binding` for `peribolos-as-service` `ServiceAccount`.

### CODE REVIEW

File: manifests/role.yaml
Comment (tumido) [inline manifests/role.yaml:10]: I think according to the specification this should reference the plural name of the resource

```suggestion
  - "taskruns"
```

image

File: manifests/role.yaml
Comment (tumido) [inline manifests/role.yaml:5]: Let's leave the namespace to be filled in by kustomize in here

File: manifests/role-binding.yaml
Comment (tumido) [inline manifests/role-binding.yaml:5]: Let's leave the namespace to be filled in by kustomize in here

File: manifests/role-binding.yaml
Comment (tumido) [inline manifests/role-binding.yaml:9]: Kustomize can also change this value. Let's leave the namespace here to be filled in as well...

File: manifests/role.yaml
Comment (SamoKopecky) [reply manifests/role.yaml:10]: Yes it should thanks!

**Comment by tumido** [comment]
/hold

Please don't recreate the PR, rather fix your history and force push. :slightly_smiling_face: 

I think a simple `git rebase main` should do the trick now and you should be left with 3 commits in this PR, then just `push --force` to this branch

**Review by tumido — COMMENTED** [review]
Code looks good, I like it. I'll approve once rebased :slightly_smiling_face:

**Comment by SamoKopecky** [comment]
Thanks for the tip! It should be fine now.

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**
